jetty/jetty11/src/main/java/com/linecorp/armeria/server/jetty/JettyServiceBuilder.java

```diff
@@ -0,0 +1,348 @@
+/*
+ * Copyright 2016 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.linecorp.armeria.server.jetty;
+
+import static java.util.Objects.requireNonNull;
+
+import java.util.EventListener;
+import java.util.List;
+import java.util.Map;
+import java.util.concurrent.locks.ReentrantLock;
+import java.util.function.Consumer;
+import java.util.function.Function;
+
+import org.eclipse.jetty.server.Handler;
+import org.eclipse.jetty.server.HttpConfiguration;
+import org.eclipse.jetty.server.RequestLog;
+import org.eclipse.jetty.server.Server;
+import org.eclipse.jetty.server.SessionIdManager;
+import org.eclipse.jetty.server.handler.HandlerWrapper;
+
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.ImmutableMap;
+
+import com.linecorp.armeria.common.annotation.Nullable;
+import com.linecorp.armeria.common.util.BlockingTaskExecutor;
+
+/**
+ * Builds a {@link JettyService}. Use {@link JettyService#of(Server)} if you have a configured Jetty
+ * {@link Server} instance.
+ */
+public final class JettyServiceBuilder {
```

Comment:
How about adding `AbstractJettyServiceBuilder` to share the common part with `JettyServiceBuilder` in `jetty9`?

Reply:
That's a good suggestion. Addressed. 👍